### DISCUSSION TITLE
more than one K-Type Sensor

### DISCUSSION THREAD

**Discussion by 001oc** [arendst/Tasmota#18635] [Q&A] [answered]
Hi together,

I have a use case for having more than one K-Type Sensor based on the MAX 31855 implementation of tasmota connected to an ESP32. Is it possible to put more than one of this sensors to an ESP32? I have tried it and failed till now. I have connected two MAX 31855 to completely different ports (not solely the CS pin). I can switch between the ports / k-typesensors but can not run them similtaniously together.

Is there anyone the has run at least two k-type sensors and give me a hand how it is working.

Thanks in advance

Michael

**Comment by 001oc** [comment, marked as answer]
I have just seen that was a thread 3 years ago and has not been implemented yet.

**Reply by sfromis** [reply]
You may be interested in the detail that with MAX31865 instead, Tasmota does AFAICT have support for up to 6. To be clear, as I've not worked with that one either, I'm not 100% sure of details, but I do notice that up to 6 CS pins can be selected for that module, and the reporting via JSON and web UI also includes that number.

**Reply by 001oc** [reply]
Thanks for the hint. I have tried it using 3 io-ports for each sensor - which is not functioning right now, I believe. I will try it using solely different cs for each sensor.
